waterlog/src/Pages/Map.js

```diff
@@ -1,10 +1,11 @@
 import React, {Component} from 'react';
+import HistoryComponent from './History';
 
 class MapComponent extends Component {
   render() {
     return ( 
     <div className = "App" >
-      <div> Map Component </div> 
+      <div> <HistoryComponent id={2}/> </div> 
```

Comment:
why 2?

Reply:
Are you asking why the component History Component is nested inside the Map Component, or about the hard coded ID?

Reply:
Never mind the first comment, I had missed the 2 in your question.

For testing purposes, once the page is linked to a page which belongs to a pending feature, will use props to pass it the id

Reply:
Okay, but I think you should avoid pushing static test data `might cause unnecessary issues` well this was just out of curiosity so ...as you were.